docs/sources/operators-guide/configure/about-configurations.md

```diff
@@ -17,6 +17,36 @@ A given configuration loads at startup and cannot be modified at runtime. Howeve
 
 To see the current configuration state of any component, use the [`/config`]({{< relref "../reference-http-api/index.md#configuration" >}}) or [`/runtime_config`]({{< relref "../reference-http-api/index.md#runtime-configuration" >}}) HTTP API endpoint.
 
+## Common configurations
+
+Some configurations, such as object storage backend, are repeated for multiple components.
+To avoid the repetition in the configuration file, use the [`common`]({{< relref "../configure/reference-configuration-parameters/index.md#common" >}}) configuration section or `-common.*` CLI flags.
+Common configurations are first applied to all the specific configurations, which allows them to be overridden later by specific values.
+
+For example, the following configuration uses the same Amazon S3 object storage bucket called `mimir`. The common storage is located in the `us-east` region for both ruler and alertmanager stores, and the blocks storage uses the `mimir-blocks` bucket from the same region:
+
+```yaml
+common:
+  storage:
+    backend: s3
+    s3:
+      region: us-east
+      bucket_name: mimir
+
+blocks_storage:
+  s3:
+    bucket_name: mimir-blocks
+```
+
+For a reference of this configuration, see [object storage configuration reference ]({{< relref "configure-object-storage-backend.md" >}}).
+
+The precedence of the common configuration is as follows, where each configuration overrides the previous one:
+
+- YAML common values
+- YAML specific values
+- CLI common flags
+- CLI specific flags
+
 ## Operational considerations
 
 Use a single configuration file, and either pass it to all replicas of Grafana Mimir (when running multiple single-process Mimir replicas) or to all components of Grafana Mimir (when running Grafana Mimir as microservices). When running Grafana Mimir on Kubernetes, you can achieve this by storing the configuration file in a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) and mounting it in each Grafana Mimir container.
```

Comment:
```suggestion
Use a single configuration file, and either pass it to all replicas of Grafana Mimir (if you are running multiple single-process Mimir replicas) or to all components of Grafana Mimir (if you are running Grafana Mimir as microservices). If you are running Grafana Mimir on Kubernetes, you can achieve this by storing the configuration file in a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) and mounting it in each Grafana Mimir container.
```
Vet for technical accuracy after my changes.

Does "when" mean "while" because something is happening simultaneously, or does it mean "if"? Review each instances of "when" and ask the same question.

Reply:
Yes, the change makes sense, it's an _if_.